src/client/src/ui/shell/views/shell.scss

```diff
@@ -6,6 +6,11 @@ $workspace-height:            calc(100vh - #{$header-height} - #{$blotter-height
 $analytics-width:             400px;
 $workspace-width:             calc(100vw - #{$analytics-width});
 
+@font-face {
+  font-family: 'Rubik Light';
+  src: url('../../../resources/fonts/RubikLight.woff2');
+}
+
```

Comment:
Google translate doesn't do so well here.

On 14 April 2016 at 17:04, Keith Woods <notifications@github.com> wrote:

> In src/client/src/ui/shell/views/shell.scss
> <https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/247#discussion_r59744122>
> :
>
> > @@ -6,6 +6,11 @@ $workspace-height:            calc(100vh - #{$header-height} - #{$blotter-height
> >  $analytics-width:             400px;
> >  $workspace-width:             calc(100vw - #{$analytics-width});
> >
> > +@font-face {
> > +  font-family: 'Rubik Light';
> > +  src: url('../../../resources/fonts/RubikLight.woff2');
> > +}
> > +
>
> 不會
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/247/files/f37ec29f1cfad02adfc2da8830327da7912b0dea#r59744122>
>

-- 
 

*This email and any attachments to it may be confidential and are intended 
solely for the use of the individual to whom it is addressed. Any views or 
opinions expressed are solely those of the author and do not necessarily 
represent those of Adaptive Financial Consulting Ltd. *

*If you are not the intended recipient of this email, you must neither take 
any action based upon its contents, not copy nor show it to anyone.*

*Please contact the sender if you believe you have received this email in 
error.*

*Adaptive Financial Consulting Ltd is a company registered in England and 
Wales. Registered number: 08105955. Registered office: 2nd floor, 11 
Leadenhall Street, London EC3 V1LP.*